src/libraries/restore/binplacePackages/binplacePackages.depproj

```diff
@@ -37,6 +37,7 @@
   
   <ItemGroup>
     <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="1.0.0" Condition="'$(TargetsNetStandard)' != 'true' or '$(NETStandardVersion)' &gt;= 2.0" />
+    <PackageReference Include="System.Data.SqlClient" Version="4.8.0" Condition="'$(TargetFramework)' == '$(NetCoreAppCurrent)'" />
```

Comment:
This is the right place, but I believe you'll also need it for netstandard https://github.com/dotnet/runtime/blob/23c54f183c9faf2049223a44197c6b8b282b90c4/src/libraries/System.Data.OleDb/src/Configurations.props#L5.